config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
Re: DATABASE_URL

I'm trying similar change on my side, when I run the following:

```
$ env DATABASE_URL="mysql://USER:PASSWORD@IPADDRESS/DBNAME" bin/cake server
```

I'm getting the following warning:
```
Notice Error: Undefined index: className in [(omitted)...vendor/cakephp/cakephp/src/Datasource/ConnectionManager.php, line 116]
```

and can not connect to the DB.

What's the right syntax for the DATABASE_URL?

Also do you have a plan to add tests for it?